nfl_server/public/scatter_plot.js

```diff
@@ -244,19 +249,23 @@ const updatePlot = () => {
 	updateScatterPlotYValues(checkedAttributes, sliderAttributes);
 };
 
-let data, qbData, wrData;
+let data, qbData, wrData, rbData;
 let shownPlayers = [];
 Promise.all([
 	d3.csv('http://localhost:3000/data/career_passing_stats_10'),
-	d3.csv('http://localhost:3000/data/career_receiving_stats_10')
+	d3.csv('http://localhost:3000/data/career_receiving_stats_10'),

```

Comment:
Whoops I thought I changed these and I guess it is still working on Heroku, but `/data/career_receiving_stats_10` should also work